framework/include/loops/ComputeFVFluxThread.h

```diff
@@ -396,16 +397,27 @@ ComputeFVFluxThread<RangeType>::onBoundary(const FaceInfo & fi, BoundaryID bnd_i
       .template condition<AttribVectorTags>(_tags)
       .template condition<AttribBoundaries>(bnd_id)
       .queryInto(bcs);
-  if (bcs.size() == 0)
-    return;
-
-  reinitVariables(fi);
```

Comment:
We dont need to re-init on boundaries anymore because it is always done onFace right

Reply:
correct